nginx-pckg-module/src/media/codec_config.c

```diff
@@ -492,12 +496,38 @@ codec_config_mp4a_config_parse(
 
     if (reader.stream.eof_reached)
     {
-        vod_log_error(VOD_LOG_ERR, log, 0,
-            "codec_config_mp4a_config_parse: failed to read all required audio extra data fields");
-        return VOD_BAD_DATA;
+        goto error;
+    }
+
+    if (result->object_type == AOT_SBR || result->object_type == AOT_PS )
+    {
+        uint8_t ext_sample_rate_index = bit_read_stream_get(&reader, 4);
```

Comment:
the convention here is to define all vars in the beginning of the func

Reply:
start of the block is the same